pkg/cmd/server/api/types.go

```diff
@@ -296,19 +296,19 @@ type MasterConfig struct {
 
 // JenkinsPipelineConfig holds configuration for the Jenkins pipeline strategy
 type JenkinsPipelineConfig struct {
```

Comment:
Oh, the reason is so that we can enable by default but admins can still disable.  If we make the parent a pointer, we'd still have to have the Enabled flag.  Since we have to have the enabled flag, would prefer not to have a pointer.  This is a core part of the system.